RecoEgamma/EgammaIsolationAlgos/interface/EgammaTowerIsolation.h

```diff
@@ -220,7 +220,9 @@ class EgammaTowerIsolation {
 			float etLow,
 			signed int depth,
 			const CaloTowerCollection* towers );
-  
+
+
+  void setTLS(const CaloTowerCollection* towers);  
```

Comment:
If this method is not needed anywhere other than in the constructor, just fold it back to the constructor.